deepin-clone.pro

```diff
@@ -1,3 +1,4 @@
+PREFIX = /usr
```

Comment:
the "PREFIX" d'not defined in this, Doing so will not make it valid in other pro files.
Can move to the "common.pri" file
````
isEmpty(PREFIX): PREFIX = /usr
````